src/components/common/ToastMessage.jsx

```
@@ -0,0 +1,15 @@
+const ToastMessage = ({ children }) => (
```

ESLint

[error] 'children' is missing in props validation

(react/prop-types)